src/renderer/components/monaco-editor/monaco-editor.tsx

```diff
@@ -37,11 +39,13 @@ export interface MonacoEditorProps {
   onDidContentSizeChange?(evt: editor.IContentSizeChangedEvent): void;
   onModelChange?(model: editor.ITextModel, prev?: editor.ITextModel): void;
   innerRef?: React.ForwardedRef<MonacoEditorRef>;
+  setInitialHeight?: boolean;
 }
 
 interface Dependencies {
   userStore: UserStore;
   activeTheme: IComputedValue<LensTheme>;
+  getEditorHeightFromLinesCount: MonacoEditorSize;
```

Comment:
_btw_: looks a bit misleading type name `MonacoEditorSize` for some **function/action** and looks like a `number` or set of constants / enum. Wdyt?

Reply:
Set an inline type 
```
interface Dependencies {
  userStore: UserStore;
  activeTheme: IComputedValue<LensTheme>;
  getEditorHeightFromLinesCount: (linesCount: number) => number;
}
```